cli/crates/cli/src/init.rs

```diff
@@ -1,8 +1,26 @@
-use crate::{errors::CliError, output::report};
-use backend::project;
+use crate::{cli_input::ConfigFormat, errors::CliError, output::report, prompts::handle_inquire_error};
+use backend::project::{self, ConfigType, ProjectTemplate};
+use inquire::Select;
+
+pub fn init(name: Option<&str>, template: Option<&str>, config_format: Option<ConfigFormat>) -> Result<(), CliError> {
+    let template = match (template, config_format) {
+        (Some(template), _) => ProjectTemplate::FromUrl(template),
+        (None, Some(ConfigFormat::TypeScript)) => ProjectTemplate::FromDefault(ConfigType::TypeScript),
+        (None, Some(ConfigFormat::GraphQL)) => ProjectTemplate::FromDefault(ConfigType::GraphQL),
+        (None, None) => {
+            let config_type = Select::new(
+                "In which format the project schema and configuration should be written?",
```

Comment:
```suggestion
                "In which format should the configuration should be written in?",
```

Reply:
I think it's either "what format" and "in" for "which" without "in, both "which" and "in" sound a bit weird to me.
There's the whole [ending a sentence in a preposition](https://www.scribbr.com/parts-of-speech/ending-a-sentence-with-a-preposition) debate but it's not critical